core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -52,13 +51,6 @@ class DescribeConsumerGroupTest extends ConsumerGroupCommandTest {
     }
   }
 
-  @Test(expected = classOf[OptionException])
-  def testDescribeWithMultipleSubActions(): Unit = {
-    TestUtils.createOffsetsTopic(zkClient, servers)
-    val cgcArgs = Array("--bootstrap-server", brokerList, "--describe", "--group", group, "--members", "--state")
-    getConsumerGroupService(cgcArgs)
-  }
-
```

Comment:
By reusing the `state` flag for both describe and list The combination is only checked in the `ConsumerGroupCommandOptions.checkArgs()` method - used by the cli but already not covered by the test.
test coverage of `checkArgs` could be another PR ?

Reply:
I would prefer to keep this tested somewhere in order to not reduce our test coverage. What about updating this test to call `checkArg()` and perhaps use `grabConsoleOutput`? Otherwise, let's add a similar unit test somewhere else.

Reply:
Because `checkArgs()` calls exit, it's a bit strange to test but I've readded this test